src/components/moviesList.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import MovieItem from './movieItem';
+
+class MoviesList extends Component {
+
+    render() {
+        const { movies } = this.props; 
+
+        if(movies.moviesList && movies.moviesList.Search) {
+            return (
+                <div className="movieslist-item-wrapper">
```

Comment:
BEM is a good practice to follow in CSS class naming. You can read about it here: http://getbem.com/